pages/vi/profiles/ArashDai.md

```diff
@@ -0,0 +1,22 @@
+# Arash Dai
+Los Angeles, Ca, PST Ubuntu 17.10 & Windows 10
+
+![alt text](https://avatars1.githubusercontent.com/u/8280071?s=460&v=4 "a picture of me")
```

Comment:
Could we replace `alt text` with something meaningful?

See more https://ux.shopify.com/writing-for-all-people-how-to-use-alternative-text-well-1205a18307a1